src/t8_forest/t8_forest_general.h

```diff
@@ -570,6 +570,43 @@ t8_forest_leaf_face_neighbors (t8_forest_t forest, t8_locidx_t ltreeid, const t8
                                t8_locidx_t **pelement_indices, t8_eclass_scheme_c **pneigh_scheme,
                                int forest_is_balanced);
 
+/** Like \ref t8_forest_leaf_face_neighbors but also provides information about the global neighbors and the orientation. 
+    * \param [in]    forest  The forest. Must have a valid ghost layer.
+    * \param [in]    ltreeid A local tree id.
+    * \param [in]    leaf    A leaf in tree \a ltreeid of \a forest.
+    * \param [out]   pneighbor_leaves Unallocated on input. On output the neighbor
+    *                        leaves are stored here.
+    * \param [in]    face    The index of the face across which the face neighbors
+    *                        are searched.
+    * \param [out]   dual_faces On output the face id's of the neighboring elements' faces.
+    * \param [out]   num_neighbors On output the number of neighbor leaves.
+    * \param [out]   pelement_indices Unallocated on input. On output the element indices
+    *                        of the neighbor leaves are stored here.
+    *                        0, 1, ... num_local_el - 1 for local leaves and
+    *                        num_local_el , ... , num_local_el + num_ghosts - 1 for ghosts.
+    * \param [out]   pneigh_scheme On output the eclass scheme of the neighbor elements.
+    * \param [in]    forest_is_balanced True if we know that \a forest is balanced, false
+    *                        otherwise.
+    * \param [out]   gneigh_tree  The global tree IDs of the neighbor trees.
```

Comment:
`` suggestion
  * \param [in]    forest  The forest. Must have a valid ghost layer.
  * \param [in]    ltreeid A local tree id.
  * \param [in]    leaf    A leaf in tree \a ltreeid of \a forest.
  * \param [out]   pneighbor_leaves Unallocated on input. On output the neighbor
  *                        leaves are stored here.
  * \param [in]    face    The index of the face across which the face neighbors
  *                        are searched.
  * \param [out]   dual_faces On output the face id's of the neighboring elements' faces.
  * \param [out]   num_neighbors On output the number of neighbor leaves.
  * \param [out]   pelement_indices Unallocated on input. On output the element indices
  *                        of the neighbor leaves are stored here.
  *                        0, 1, ... num_local_el - 1 for local leaves and
  *                        num_local_el , ... , num_local_el + num_ghosts - 1 for ghosts.
  * \param [out]   pneigh_scheme On output the eclass scheme of the neighbor elements.
  * \param [in]    forest_is_balanced True if we know that \a forest is balanced, false
  *                        otherwise.
  * \param [out]   gneigh_tree  The global tree IDs of the neighbor trees.
```